### DISCUSSION TITLE
Is there a way to disable plugins in the UI?

### DISCUSSION THREAD

**Discussion by mikekuzak** [danny-avila/LibreChat#1293] [Q&A] [answered]
For security reasons I would like to disable the plugin section, is that possible ? 
Thanks

**Comment by danny-avila** [comment, marked as answer]
Yes, in your .env file

> Uncomment ENDPOINTS to customize the available endpoints in LibreChat

```bash
# ENDPOINTS=openAI,gptPlugins,anthropic # before
ENDPOINTS=openAI,anthropic #after
```


source: https://docs.librechat.ai/install/dotenv.html#endpoints

**Reply by mikekuzak** [reply]
Ok .. how about the store ? 

![image](https://github.com/danny-avila/LibreChat/assets/18675587/85e360ae-ffae-4d4a-ae19-fa9440cf60f9)

**Reply by danny-avila** [reply]
not possible without making changes to the git history. Is disabling the endpoint not sufficient?

**Reply by danny-avila** [reply]
or do you mean you only want to disable specific plugins?

**Reply by mikekuzak** [reply]
So I understand the endpoint are the models to use that's fine. I have setup google and openAI. 
But is there a way to disable the plugin store or the plugin section ? 

![image](https://github.com/danny-avila/LibreChat/assets/18675587/8942c822-a3ed-4427-b682-053172f6ddfa)

**Reply by danny-avila** [reply]
@mikekuzak you're on an old version of LibreChat. The instructions I gave you before are valid with the latest changes to the repo.

**Reply by fuegovic** [reply]
> So I understand the endpoint are the models to use that's fine. I have setup google and openAI. But is there a way to disable the plugin store or the plugin section ?

Looks like you are on the old UI, first of all you'll have to update. Then like @danny-avila already explained, you need to edit this line in the .env file. (look at the .env.example for the updated .env variable, it's in the top section)

In your case, you want to add this in the .env file:
`ENDPOINTS=openAI,google`

With this new configuration applied no one will be able to use plugins
![image](https://github.com/danny-avila/LibreChat/assets/32828263/67f73106-b771-4a7b-98ad-6529030c7bd2)

**Reply by mikekuzak** [reply]
@danny-avila I'm running the [v0.6.1](https://github.com/danny-avila/LibreChat/releases/tag/v0.6.1) docker container.

**Reply by fuegovic** [reply]
@mikekuzak if you have node installed, use this command to update:
`npm run update`

There have been some improvements since the release of v0.6.1 (even tho we're still on v0.6.1)

**Reply by danny-avila** [reply]
We're due a release soon for v0.6.2
